scheduler/cups-driverd.cxx

```diff
@@ -21,7 +21,10 @@
 #include <cups/transcode.h>
 #include <cups/ppd-private.h>
 #include <ppdc/ppdc.h>
+#include <pthread.h>
```

Comment:
As the pthread code in CUPS is currently *optional*, you need to make running the driver interfaces in parallel optional (and the pthread include optional) based on the HAVE_PTHREAD preprocessor define.